go/vt/vttablet/onlineddl/executor.go

```diff
@@ -1042,6 +1042,15 @@ func (e *Executor) ExecuteWithVReplication(ctx context.Context, onlineDDL *schem
 		if _, err := e.vreplicationExec(ctx, tmClient, tablet.Tablet, insertVReplicationQuery); err != nil {
 			return err
 		}
+
+		{
+			// temporary hack. todo: this should be done when inserting any _vt.vreplication record across all workflow types
+			query := fmt.Sprintf("update _vt.vreplication set workflow_type = %d where workflow = '%s'",
+				binlogdatapb.VReplicationWorkflowType_ONLINEDDL, v.workflow)
+			if _, err := e.vreplicationExec(ctx, tmClient, tablet.Tablet, query); err != nil {
+				return vterrors.Wrapf(err, "VReplicationExec(%v, %s)", tablet.Tablet, query)
+			}
+		}
```

Comment:
This is to "upgrade" any existing workflow records, right?

Reply:
No, actually. This immediately follows an INSERT. Because I did not want to import the entirety of #9742 , the INSERT does not yet populate the new `workflow_type` column. So right after we `INSERT`, we then update the `workflow_type` column.
In the future, the above call (see `insertVReplicationQuery`) will include the columns.

So this only affects a newly submitted online ddl workflow.